tslint.json

```diff
@@ -2,6 +2,19 @@
   "extends": ["tslint:recommended"],
   "rulesDirectory": ["node_modules/codelyzer"],
   "rules": {
+    "align": {
```

Comment:
Almost all changes in this file are formatting rules and formatting is handled by Prettier in this repository, so you can revert all of them.

Reply:
Reverted this file